python/Scripts/mxcustomnamespacefixer.py

```diff
@@ -0,0 +1,57 @@
+# Fixup code namespaces to be customizable:
+# As run for MATX-341 update phase
+
+import subprocess, os
+
+mx_root = r"."
+clf_exe = r"clang-format"
+git_exe = r"git"
+
+def fix_namespaces(cur_file):
+    seen_open = False
+    num_open = 0
+    num_close = 0
+    with open(cur_file, "rt") as in_f:
+        with open(cur_file+b".fixed", "wt") as out_f:
+            for cur_line in in_f:
+                if cur_line == "namespace MaterialX\n":
+                    out_f.write("MATERIALX_NAMESPACE_BEGIN\n")
+                    num_open += 1
+                    seen_open = True
+                    continue
+                if cur_line == "{\n" and seen_open:
+                    out_f.write("\n")
+                    seen_open = False
+                    continue
+                if cur_line == "} // namespace MaterialX\n" or cur_line == "} // namespace MaterialX":
+                    out_f.write("MATERIALX_NAMESPACE_END\n")
+                    num_close += 1
+                    continue
```

Comment:
clang-format will produce a strictly uniform formatting for namespaces, so we can clearly find where the namespace begins and ends and replace those with the namespace macros.